alfresco-content-services/values.yaml

```diff
@@ -100,7 +101,7 @@ repository:
     baseSize: 20Gi
     # -- Specify a storageClass for dynamic provisioning
     accessModes:
-      - ReadWriteMany
+      - ReadWriteOnce
```

Comment:
do we fully understand the consequences of changing access mode to RWO? if not we should have a backlog ticket to investigate and determine if we can use RWO here

Reply:
yes this also needs more work so we'll add a new story in our backlog.